core/src/main/java/org/commonjava/indy/core/ctl/NfcController.java

```diff
@@ -46,7 +53,7 @@
 {
 
     @Inject
-    protected NotFoundCache cache;
+    protected AbstractNotFoundCache cache;
```

Comment:
If AbstractNotFoundCache is moved to galley, we don't need to switch away from the interface, do we? Also, is there any reason not to add the new methods to the interface itself?

Reply:
The AbstractNotFoundCache adds some methods regarding pagination which I think may not be the core function of NFC interface. Someone else may not need to render it via pages. I consider it a specific requirement for Indy UI. But I am open to suggestion here.

Reply:
We can adjust later as needed. It's fine for now.